### ISSUE TITLE
java.lang.IllegalArgumentException: already added :  Landroid/support/v4/accessibilityservice/AccessibilityServiceInfoCompat;

### ISSUE THREAD

**Issue by spurdow** [Redth/ZXing.Net.Mobile#631]
First and foremost thanks to this wonderful library. Its just made us more productive.
Im currently using zxing `2.2.9`.
After installing I cannot run the project anymore because of the error

```java
 java.lang.IllegalArgumentException: already added : Landroid/support/v4/accessibilityservice/AccessibilityServiceInfoCompat;

```

Kindly provide feedback/guidelines to accommodate my issue. Thanks

**Comment by spurdow** [comment]
Fixed just update the Support.v4 to latest

**Comment by TonyC5** [comment]
@spurdow - thanks for your solution.  This worked for me, too; however, 'update the Support.v4 to latest' was not as simple form me as I had hoped.  Sharing my experience as it might help others.  

I am using ZXing.Mobile library with Visual Studio/Xamarin Forms (PCL - Android and iOS).  My Android target framework was 7.1.  I was unable to update the Support.v4 library, because the newer version (required to resolve the problem reported in this thread) was not compatible with my target framework.  I needed to increase my target framework to 8.0 in order to update Support.v4.  I also needed to re-run Update Nuget Packages to make sure that other packages were updated and compatible.
